config/runtime/fstests.jinja2

```diff
@@ -0,0 +1,8 @@
+{# -*- mode: Python -*- -#}
+{# SPDX-License-Identifier: LGPL-2.1-or-later -#}
+
+{%- extends 'base/shell.jinja2' %}
+
+{%- block commands %}
+kvm-xfstests --help
+{% endblock %}
```

Comment:
After looking into the next steps here, I think it's really necessary to move to Python as there aren't tools available yet for submitting the results on the command line.  So this is probably fine for this PR but it will need to be removed in the next one when generating the kernel config file.

Reply:
Ok, I just drop the commit..then I can add a commit to drop in my next PR.